pycaret/internal/plots/time_series.py

```diff
@@ -1025,3 +1025,89 @@ def plot_time_series_decomposition(
 
     return fig, return_data_dict
 
+
+def plot_multiple_predictions_with_confidence(
```

Comment:
This change should apply to the plot without confidence interval as well. Can you update that as well?
Also, we still need to support confidence interval if a single forecast is provided.